packages/libs/eda/src/lib/core/components/visualizations/implementations/VolcanoPlotVisualization.tsx

```diff
@@ -460,6 +471,71 @@ function VolcanoPlotViz(props: VisualizationProps<Options>) {
     ...(data.value ? {} : EMPTY_VIZ_AXIS_RANGES),
   };
 
+  // set truncation flags
+  const {
+    truncationConfigIndependentAxisMin,
+    truncationConfigIndependentAxisMax,
+    truncationConfigDependentAxisMin,
+    truncationConfigDependentAxisMax,
+  } = useMemo(
+    () =>
+      truncationConfig(
+        {
+          independentAxisRange: {
+            min: rawDataMinMaxValues.x.min,
+            max: rawDataMinMaxValues.x.max,
+          },
+          dependentAxisRange: {
+            min: Math.min(
+              -Math.log10(rawDataMinMaxValues.y.min),
+              -Math.log10(rawDataMinMaxValues.y.max)
+            ),
+            max: Math.max(
+              -Math.log10(rawDataMinMaxValues.y.min),
+              -Math.log10(rawDataMinMaxValues.y.max)
+            ),
+          },
```

Comment:
We know that the -log(y.max) will always be less than -log(y.min), so we can simplify the dependentAxisRange to:

```suggestion
          dependentAxisRange: {
            min:  -Math.log10(rawDataMinMaxValues.y.max),
            max: -Math.log10(rawDataMinMaxValues.y.min),
          },
```

Reply:
Thanks @asizemore for your suggestion 👍  I applied your changes to the new commit, so I will merge this PR.